pkg/instance_sharing/types.go

```diff
@@ -4,11 +4,21 @@ const (
 	ReferencePlanDescription = "Allows to create a reference to a shared service instance from any environment in a subaccount and manage service bindings to that service instance."
 	ReferencePlanName        = "reference-instance"
 
-	ReferencedInstanceIDKey = "referenced_instance_id"
-
 	SupportsInstanceSharingKey = "supportsInstanceSharing"
 
-	ReferencePlanNameSelector     = "plan_name_selector"
-	ReferenceInstanceNameSelector = "instance_name_selector"
-	ReferenceLabelSelector        = "instance_labels_selector"
+	ReferencedInstanceIDKey         = "referenced_instance_id"
+	ReferencedInstanceIDTitle       = "Reference Instance ID"
+	ReferencedInstanceIDDescription = "Specify the ID of the instance to which you want to refer."
+
+	ReferenceInstanceNameSelectorKey         = "instance_name_selector"
+	ReferenceInstanceNameSelectorTitle       = "Find by instance name"
+	ReferenceInstanceNameSelectorDescription = "You can use various search criteria to find shared instances to which you want to create a reference."
+
+	ReferencePlanNameSelectorKey         = "plan_name_selector"
+	ReferencePlanNameSelectorTitle       = "Find by plan name"
+	ReferencePlanNameSelectorDescription = "You can use various search criteria to find shared instances to which you want to create a reference."
```

Comment:
Create a reference to a shared instance that was created with a specific plan. \nFor example, plan=large returns a shared instance that was created with the 'large' plan, and then you create a reference to that instance.

Reply:
You can use the plan name to find the shared instance to which you want to create a reference.